salt/modules/yumpkg.py

```diff
@@ -1056,9 +1056,10 @@ def install(name=None,
     return ret
 
 
-def upgrade(refresh=True, fromrepo=None, skip_verify=False, **kwargs):
+def upgrade(name=None, refresh=True, fromrepo=None, skip_verify=False, pkgs=None, normalize=True, **kwargs):
```

Comment:
@stardust85, the issue is that there may be circumstances where users are using this function with arguments as a list, `pkg.upgrade(False)` rather than a dictionary `pkg.upgrade(refresh=False)` and reordering the argument list would cause unintended behavior in that case.  Another consideration is that modules that share the same `__virtualname__` should try to maintain API parity as much as possible.